service/src/main/java/bio/terra/catalog/common/DaoKeyHolder.java

```diff
@@ -19,10 +19,7 @@ public Timestamp getTimestamp(String fieldName) {
 
   public Instant getCreatedDate() {
     Timestamp timestamp = getTimestamp("created_date");
-    if (timestamp != null) {
```

Comment:
Why can we do this? Is `created_date` non nullable?

Reply:
this is because timestamp can never produce a null value

Reply:
Yes, the column has a `nullable: false` constraint.